virttest/utils_hotplug.py

```diff
@@ -23,8 +23,8 @@
 
 def create_mem_xml(tg_size, pg_size=None, mem_addr=None, tg_sizeunit="KiB",
                    pg_unit="KiB", tg_node=0, node_mask=0, mem_model="dimm",
-                   lb_size=None, lb_sizeunit="Kib", mem_access=None,
-                   alias=None):
+                   mem_discard="no", alias=None, lb_size=None, lb_sizeunit="Kib",
```

Comment:
suggest put the newly added parameter into last position to avoid mismatch parameter in case value is given without key=value mode

Reply:
I have checked the whole code. There is no mismatch. If I put it in the last position, some script need to change